src/components/ActionBar/ActionBarContent.tsx

```diff
@@ -8,21 +9,26 @@ export interface ActionBarContentProps {
   children?: React.ReactNode;
 }
 
-export const ActionBarContent = ({ children }: ActionBarContentProps) => {
-  const { open, contentId, headerId, color } = useActionBarContext();
+export const ActionBarContent = forwardRef<HTMLDivElement, ActionBarContentProps>(
+  ({ children }, ref) => {
```

Comment:
You can pick prop children too. Even though you haven't done anything with the props here. But would be nice to get that change in this PR.